packages/imagemagick7.rb

```diff
@@ -3,7 +3,7 @@
 class Imagemagick7 < Autotools
   description 'Use ImageMagick to create, edit, compose, or convert bitmap images.'
   homepage 'http://www.imagemagick.org/script/index.php'
-  version "7.1.1-36-perl5.40-#{CREW_ICU_VER}"
+  version "7.1.2-1-perl5.40-#{CREW_ICU_VER}"
```

Comment:
Drop the `CREW_ICU_VER` tag now that the package no longer depends on it?

Reply:
This is done.